web/src/hotkey.js

```diff
@@ -270,13 +270,12 @@ export function process_escape_key(e) {
         return true;
     }
 
-    if (popovers.any_active() || sidebar_ui.any_sidebar_expanded_as_overlay()) {
+    if (popovers.any_active()) {
```

Comment:
This commit had the phrase "This also helps fix #26717, because ...". I've reworded that to say "helps with ..." to avoid auto-closing #26717 from GitHub's silly regexs.